gdpr/purposes/default.py

```diff
@@ -19,6 +19,9 @@ def __new__(mcs, name, bases, attrs):
             purposes_map[new_class.slug] = new_class
         return new_class
 
+    def __str__(self):
```

Comment:
Proč je to v metaclasse?

Reply:
Protože se třídy dědící z AbstractPurpose nevytváří objekty. A stringová reprezentace pro classu se definuje na metaclasse.